core/src/main/scala/org/apache/spark/SparkHadoopWriter.scala

```diff
@@ -143,6 +141,12 @@ class SparkHadoopWriter(jobConf: JobConf)
     taskContext
   }
 
+  protected def newTaskAttemptContext(
```

Comment:
Yea, it's overridden in a mock object in `OutputCommitCoordinatorSuite` so I kept this